project/src/components/not-found/not-found.tsx

```diff
@@ -0,0 +1,14 @@
+import {Fragment} from 'react';
+import {Link} from 'react-router-dom';
+
+function NotFoundScreen(): JSX.Element{
+  return (
+    <Fragment>
+      <h1>404. Page not found</h1>
+      <hr />
+      <Link to='/'>Back to main page</Link>
```

Comment:
```suggestion
      <Link to={AppRoute.Main}>Back to main page</Link>
```
У тебя есть енам. Используй его по максимуму.

Reply:
упустил))